README.md

```diff
@@ -121,6 +127,14 @@ More information can be found by reading our [.travis.yml](.travis.yml) file.
 - GNU ARM Embedded toolchain
 - GNU Make 4.0 or above
 - [Unity&mdash;Throw the Switch](http://www.throwtheswitch.org/unity/): our C unit testing framework
+- Go 1.10.4 or above 
+  - If your Go version is out of date (which it probably will be if you installed the default box) you can update it via the [update-golang](https://github.com/udhos/update-golang) script, preferably outside the current repository (e.g in your vagrant box shared folder)
+    ```
+    cd ..
+    git clone https://github.com/udhos/update-golang
+    cd update-golang
+    sudo ./update-golang.sh
+    ```
```

Comment:
@jessm what do you think about ditching generating for Go entirely so we don't have to deal with this since we're not using it? Is it valuable to keep maintaining it just in case?

Reply:
I'm all in favour of throwing it out. "Maintaining" is a pretty strong word in a student design team, I think the less code we keep the better off we are.

Reply:
Just to confirm: we're throwing out the Go toolchain entirely?

Reply:
Protobuf generation is the only thing we're using the Go toolchain for right? I'd say just throw out the whole thing since we aren't using it and it's less to maintain. I'd also say to ditch the TypeScript generation since we're also not using that (unless @AveryChiu64 you have a use for it in CAN explorer/telemetry/driver display?).